apps/web/src/components/UserProfile/ExperienceSection.tsx

```diff
@@ -194,7 +194,24 @@ const ExperienceSection = ({
               />
             ))}
           </Accordion.Root>
-        ) : null}
+        ) : (
+          <div
+            data-h2-background-color="base(background.dark)"
+            data-h2-border="base(1px solid background.darker)"
+            data-h2-padding="base(x1)"
+            data-h2-radius="base(s)"
+          >
```

Comment:
I think we could use the `Well` component here and it would look similar enough.

Reply:
I was hoping to use the `Well` component but it does not match the other null state for the same group which is not a component in of itself. 🤷 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/63d90c822f8eaf01439d79b2fb9e2b553e24ef95/apps/web/src/components/UserProfile/ExperienceSection.tsx#L218-L223

## Screenshot

<img width="742" alt="Screen Shot 2024-05-09 at 12 42 53" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/01a2086c-d046-4c82-a225-aeae56963355">

Reply:
`Well` component replaces these custom `div`s in [e55b417](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10336/commits/e55b417efb77f5395309d755d66164dc7e4d002e).